app/hooks/use-toast.ts

```diff
@@ -6,13 +6,23 @@ import * as React from 'react';
 import type { ToastActionElement, ToastProps } from '~/components/ui/toast';
 
 const TOAST_LIMIT = 1;
-const TOAST_REMOVE_DELAY = 1000000;
```

Comment:
I don't get it. What is remove delay and why was it this big? are these milliseconds? this is 16 minutes